src/main/java/duke/command/DeadlineCommand.java

```diff
@@ -0,0 +1,22 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.Deadline;
+
+public class DeadlineCommand extends Command {
```

Comment:
maybe can add javadoc to say what each command subclass is for for better clarity!

Reply:
Good point! I'll add it in, thank you!